src/client/pkg/require/require.go

```diff
@@ -261,6 +262,7 @@ func fatal(tb testing.TB, userMsgAndArgs []interface{}, msgFmt string, msgArgs .
 	tb.Helper()
 	logMessage(tb, userMsgAndArgs)
 	_, file, line, ok := runtime.Caller(2)
+	debug.PrintStack()
```

Comment:
This seems like a nice change, do you think it makes sense to not log the file and line redundantly below?

Reply:
I actually left this in on accident (when I was trying to debug a test failure that wasn't extracting the caller correctly because it was used inside a `defer`) but if you like it then sure, I'll get rid of the file/line log